loggo/loggo.py

```diff
@@ -34,7 +34,25 @@ class Loggo:
     # Only log when log level is this or higher
     log_threshold = logging.DEBUG
 
-    def __init__(self, config: Optional[Dict] = None) -> None:
+    def __init__(self,
+                 called: Optional[str] = FORMS['called'],
+                 returned: Optional[str] = FORMS['returned'],
+                 returned_none: Optional[str] = FORMS['returned_none'],
+                 errored: Optional[str] = FORMS['errored'],
+                 error_level: int = 50,
+                 facility: str = 'loggo',
+                 ip: Optional[str] = None,
+                 port: Optional[str] = None,
+                 do_print: bool = False,
+                 do_write: bool = False,
+                 truncation: int = 7500,
+                 raise_logging_errors: bool = False,
+                 logfile: Optional[str] = './logs/logs.txt',
```

Comment:
I think we can simplify and say this is strictly of type `str`. No need to set `None`s here as logfile is turned on/off with the boolean `do_write` anyway

Reply:
k